### ISSUE TITLE
Serverlink uses a PORT file

### ISSUE THREAD

**Issue by bleucitron** [anthill/6bin#45]
This is so very wrong, the port is now passed to the server as parameter through 6brain. It works for now because the `PORT.js` file still exists due to previous versions (wasn't deleted), and because we haven't changed the port. But this might be a problem in the future.

https://github.com/anthill/6bin/blob/master/ts/client/serverLink.ts#L8

We might need to go back to hardcoding the `PORT` for everybody, not passing it as a parameter.

**Comment by DavidBruant** [comment]
Ideally, both the client and server find the port in the same file, so that they'll always be in sync.

**Comment by DavidBruant** [comment]
But hardcoding on both end works too. The client/server divide for 6bin is just a sad necessity.

**Comment by bleucitron** [comment]
Yep, it was in a unique file before, but we changed it to be a parameter of the server... Guess ill just change it back. Thx

**Comment by DavidBruant** [comment]
Why doesn't the client just read it via `location.port`?

**Comment by bleucitron** [comment]
Because it's the first time i've heard of it :p
